pandas/_libs/window/indexers.pyx

```diff
@@ -11,7 +11,7 @@ def calculate_variable_window_bounds(
     int64_t num_values,
     int64_t window_size,
     object min_periods,  # unused but here to match get_window_bounds signature
-    object center,  # unused but here to match get_window_bounds signature
+    object center,
```

Comment:
Should be `bint center` now if supported

Reply:
thanks for your quick response, this is done now. :)